src/Longitude.cpp

```diff
@@ -71,8 +78,14 @@ const char *Longitude::formatString(char *targetBuffer, const char *format, long
 {
     long secs = totalSeconds;
 
-    // Map to 0..360 westwards
+#if USE_OLD_ASCOM_DRIVER_COMPATIBLE_PROTOCOL == 1
+    // from indilib driver:  Meade classic handset defines longitude as 0 to 360 WESTWARD (https://github.com/indilib/indi/blob/1b2f462b9c9b0f75629b635d77dc626b9d4b74a3/drivers/telescope/lx200driver.cpp#L1019)
     secs = 180L * 3600L - secs;
+#else
+    // from indilib driver: Meade API expresses East Longitudes as negative, West Longitudes as positive. (https://github.com/indilib/indi/blob/8beeeb9c5809063929804b5066f5b8ba6696bcd0/drivers/telescope/lx200driver.cpp#L1180)
+    // Source: https://www.meade.com/support/LX200CommandSet.pdf from 2002 at :Gg#
+    secs = -secs;
```

Comment:
This "solution" is broken since formatStringImpl will break on non-positive inputs to degs, mins, secs.

Reply:
"Fixed" in latest commit. I say "fixed" since it is very ugly. I am more looking forward to the changes that @ClutchplateDude are doing in the `meade-fix` branch.